changelogs/fragments/7564-onepassword-lookup-case-insensitive.yaml

```
@@ -0,0 +1,4 @@
+bugfixes:
+  - >-
+    onepassword lookup plugin - field and section titles are now case insensitive when using
+    op CLI version two or later (https://github.com/ansible-collections/community.general/pull/7564)
```

How about mentioning that this adjusts behavior to the way it works with op CLI version 1?
```suggestion
    op CLI version two or later. This was already the case when using op CLI version one
    (https://github.com/ansible-collections/community.general/pull/7564)
```